packages/bsky/src/api/app/bsky/actor/getSuggestions.ts

```diff
@@ -29,6 +29,7 @@ export default function (server: Server, ctx: AppContext) {
       const hydrateCtx = await ctx.hydrator.createContext({ viewer, labelers })
       const headers = noUndefinedVals({
         'accept-language': req.headers['accept-language'],
+        'x-bsky-topics': String(req.headers['x-bsky-topics'] || ''),
```

Comment:
The array/string casting is pretty cheeky!  Part of me loves it, but sadly I think it might be worth being explicit.
```suggestion
        'x-bsky-topics': Array.isArray(req.headers['x-bsky-topics'])
          ? req.headers['x-bsky-topics'].join(',')
          : req.headers['x-bsky-topics'],
```